system/languages/en.php

```diff
@@ -291,6 +292,11 @@
 		"ml-please-confirm-email" => "Please confirm your subscription by clicking this link",
 		"ml-confirmation-email-subject" => "Please confirm your subscription",
 		"ml-unsubscribe" => "unsubscribe",
+		"ml-send-reminder" => "remind",
+		"ml-reminder-sent" => "reminder sent",
+		"ml-reminded" => "reminded",
```

Comment:
Maybe this should be "already reminded" to emphasize that reminding can only be done once?

Reply:
done